### PR TITLE
Fixed issue #5463

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#5463

## How To Test This?
Add a filter to the product or category page and then click on clear all

### CODE REVIEW

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue [hunk 1]
```diff
@@ -28,6 +29,11 @@ export default {
     methods: {
         removeFilter(filter) {
             this.$emit('onRemoveFilter', { data: { filter } });
+        },
+
+        removeAllFilters() {
+            window.localStorage.removeItem('datagridsInfo');
+            location.reload();
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue:36]: Page should not be refresh.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue [hunk 2]
```diff
@@ -9,6 +9,7 @@
                 @onRemoveFilter="removeFilter(filter)"
             ></datagrid-filter-tag>
         </template>
+        <p @click="removeAllFilters" v-if="filters.length > 0">Clear All</p>
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue:12]: This should be inside the template and Clear should be translatable.

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue [hunk 3]
```diff
@@ -464,6 +465,12 @@ export default {
                 this.refresh();
             }
         },
-    },
+        
+        clearAllFilters() {
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue:469]: Place this method after `removeFilter`.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by akathuria289** [comment]
Fixed

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
there is **no need** for the **clear all**  option If there is **only one applied filer**, please find the screenshot [Hee(https://prnt.sc/1-S2CPrRcT1S)
And the color of the **clear all** text should be **blue.**

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Pull request merged**